be/src/olap/reader.h

```diff
@@ -201,10 +202,10 @@ class TabletReader {
     std::vector<bool> _is_lower_keys_included;
     std::vector<bool> _is_upper_keys_included;
     // contains condition on key columns in agg or unique table or all column in dup tables
-    Conditions _conditions;
+    std::unique_ptr<Conditions> _conditions;
```

Comment:
why change unique_ptr， need more `malloc` operator

Reply:
because I change tablet schema as a parameter in Conditions constructor and remove set tablet schema method because the condtion should always have a tablet schema and should not be changed. But tablet schema is not required in reader's constructor.